control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -3338,14 +3338,24 @@ func (r *HostedControlPlaneReconciler) reconcileCloudControllerManager(ctx conte
 	case hyperv1.KubevirtPlatform:
 		// Create the cloud-config file used by Kubevirt CCM
 		ccmConfig := kubevirt.CCMConfigMap(hcp.Namespace)
+
+		r.Log.Info("creating kubevirt cloud-config ConfigMap")
 		if _, err := createOrUpdate(ctx, r, ccmConfig, func() error {
-			return kubevirt.ReconcileCloudConfig(ccmConfig, hcp)
+			var infraClusterNamespace string
+			var infraKubeconfigPath string
+			if hcp.Spec.Platform.Kubevirt.InfraNamespace != "" {
+				infraClusterNamespace = hcp.Spec.Platform.Kubevirt.InfraNamespace
+				infraKubeconfigPath = "/etc/kubernetes/infra-kubeconfig/kubeconfig"
+			}
```

Comment:
all this can be moved into the `kubevirt.ReconcileCloudConfig()` function i believe.